### PR TITLE
Set apply_limiter default to false

### PR DESCRIPTION
**Pull request by szy21**
## Purpose 

Set `apply_limiter` default to false as that is most of the case. Also fixes some missing `apply_limiter` argument when converting cli to yaml. cc @valeriabarra @nefrathenrici 

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

File: config/default_configs/default_perf.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 moist: "equil"
+apply_limiter: true
```
Comment (valeriabarra) [inline config/default_configs/default_perf.yml:2]: Why do we want to use limiters in the perf jobs?

File: config/perf_configs/flame/perf_gw.yml [hunk 2]
```diff
@@ -15,3 +15,4 @@ precip_model: "0M"
 alpha_rayleigh_uh: 0.0
 job_id: "flame_perf_gw"
 moist: "equil"
+apply_limiter: true
```
Comment (valeriabarra) [inline config/perf_configs/flame/perf_gw.yml:18]: same as above

File: config/default_configs/default_perf.yml (hunk 1 above)
Comment (szy21) [reply config/default_configs/default_perf.yml:2]: Oh, I just want to keep it the same as before (We probably want to check the performance of the limiter?). But we don't have to.

File: config/perf_configs/flame/perf_gw.yml (hunk 2 above)
Comment (szy21) [reply config/perf_configs/flame/perf_gw.yml:18]: This one can be removed I think.

File: config/default_configs/default_perf.yml (hunk 1 above)
Comment (szy21) [reply config/default_configs/default_perf.yml:2]: @nefrathenrici @charleskawczynski Do we want this to be true or false in perf?

Comment (nefrathenrici) [reply config/default_configs/default_perf.yml:2]: @charleskawczynski will know for sure, but I assume we should keep it the same for consistency.

File: config/default_configs/default_config.yml [hunk 3]
```diff
@@ -182,7 +182,7 @@ trunc_stack_traces:
   value: true
 apply_limiter:
   help: "Apply a horizontal limiter to every tracer [`true` (default), `false`]"
```
Comment (nefrathenrici) [inline config/default_configs/default_config.yml:184]:
```suggestion
  help: "Apply a horizontal limiter to every tracer [`false` (default), `true`]"
```

**Review by nefrathenrici — COMMENTED** [review]
Thanks for taking care of this!

File: config/default_configs/default_perf.yml (hunk 1 above)
Comment (valeriabarra) [reply config/default_configs/default_perf.yml:2]: Charlie is out now and he will be all next week. So I guess for now we want to keep it the same, and then when we have some clarity on why it was on for the perf jobs, we can always change it, if needed.

Comment (szy21) [reply config/default_configs/default_perf.yml:2]: Sounds good to me, thanks!

**Review by valeriabarra — APPROVED** [review]

**Pull request merged**